Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -716,11 +716,9 @@ class ScrollView extends React.Component<Props, State> {
   UNSAFE_componentWillMount() {
     this._scrollResponder.UNSAFE_componentWillMount();
     this._scrollAnimatedValue = new AnimatedImplementation.Value(
-      this.props.contentOffset ? this.props.contentOffset.y : 0,
-    );
-    this._scrollAnimatedValue.setOffset(
-      this.props.contentInset ? this.props.contentInset.top || 0 : 0,
+      this.props.contentOffset?.y ?? 0,
```

Comment:
Since there is no FlowFixMe here, my guess is there is a FlowFixMe somewhere in its usage and I'd rather we fix it there than mask the error by providing a default?

Reply:
The crash was happening for me in a third party library that did not use flow / typescript. I'm also fine keeping the behavior as is and fixing the library only, the goal of this is mostly to keep the same runtime behavior as before.